fingertest:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint1
    pid: {p: 5000.0, i: 0.01, d: 500.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint2
    pid: {p: 5000.0, i: 0.01, d: 500.0}
  joint3_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint3
    pid: {p: 5000.0, i: 0.01, d: 500.0}


